packages/libs/react-ui/src/components/Tooltip/Tooltip.css.ts

```diff
@@ -0,0 +1,46 @@
+import { sprinkles, vars } from '../../styles';
+
+import { style } from '@vanilla-extract/css';
+
+export const container = style([
+  sprinkles({
+    display: 'flex',
+    placeItems: 'center',
+    gap: '$2',
+    borderRadius: '$md',
+    paddingX: '$4',
+    paddingY: '$3',
+    border: 'none',
+    fontSize: '$base',
+    backgroundColor: '$neutral1',
+    color: '$neutral6',
+    width: 'max-content',
+  }),
+  {
+    position: 'absolute',
+    top: '50%',
+    marginRight: '-50%',
+    border: `${vars.borderWidths.$md} solid ${vars.colors.$neutral2}`,
+    display: 'none',
```

Comment:
sprinkle?

Reply:
For `position` and `display` I agree that they should be turned into sprinkles, not so sure that it's worth adding the 50% and -50% rules to `top` and `marginRight`, as they seem like less common values to be used in other designs